### PR TITLE
Document package::{add,remove} and modules

### PR DESCRIPTION
**Pull request by mavam**
This PR documents modules along with `package::add` and `package::remove`.

### CODE REVIEW

**Comment by mavam** [comment]
Note that I opted for using `# add` as page title instead of `# package::add` to ensure clean breadcrumbs, as shown in this screenshot. The page hierarchy of `/package/add` in the URL also makes more sense this way.

Screenshot 2024-11-10 at 11 40 06

File: web/docs/tql2/language/modules.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+TQL features namespacing in the form of *modules* to group related
+[operators](../operators.md) and [functions](../functions.md).
+
+A *module* separates the contained operation by two colons. For example, the
+[`package::add`](../operators/package/add.md) and
+[`package::remove`](../operators/package/remove.md) operators are in the
+`package` module.
+
+Modules are not hierarchical and have exactly one level of nesting. That is,
+the language does not support constructs of the form `a::b::c`.
```
Comment (dominiklohmann) [inline web/docs/tql2/language/modules.md:12]: I think we can just leave this information out. It makes no practical difference to users.

**Review by dominiklohmann — APPROVED** [review]

File: web/docs/tql2/language/modules.md (hunk 1 above)
Comment (mavam) [reply web/docs/tql2/language/modules.md:12]: Hm, it was literally the first question I asked when I wanted to learn more about modules. To be fair, I'm not the ICP, but does it do harm?

Comment (dominiklohmann) [reply web/docs/tql2/language/modules.md:12]: I think it's useless information for users that are not developers and that docs should be as concise as possible.

Comment (mavam) [reply web/docs/tql2/language/modules.md]: Okay, removed in e492eb2.

Comment (jachris) [reply web/docs/tql2/language/modules.md]: FWIW, the language _does_ support multiple levels of nesting (not tested, but that would be a bug otherwise).

**Pull request merged**